queue_fetcher/utils/sqs.py

```diff
@@ -95,7 +95,11 @@ def send_message(queue, message):
         logger.info('New message on queue %s: %s',
                     queue.name, json.dumps(message))
     else:
-        if not isinstance(message, six.string_types):
+        is_text = (
+            isinstance(message, six.string_types)
+            or isinstance(message, bytes)
```

Comment:
Isn't `six.string_types` just `(str, bytes)` in Python 3 (or `(unicode, str)` in Python 2 where `str == bytes`)

Reply:
`isinstance(b'[]', six.string_types)` returns `False` under py3 at least

Reply:
this won't work under python 2 now that you mention it